kong/config_service.go

```diff
@@ -13,7 +12,8 @@ type AbstractConfigService interface {
 	// API endpoint using the provided reader which should contain the JSON
 	// serialized body that adheres to the configuration format specified at:
 	// https://docs.konghq.com/gateway/latest/production/deployment-topologies/db-less-and-declarative-config/#declarative-configuration-format
-	ReloadDeclarativeRawConfig(ctx context.Context, config io.Reader, checkHash bool) error
+	// It returns the response body and an error, if it encounters any.
+	ReloadDeclarativeRawConfig(ctx context.Context, config io.Reader, checkHash bool) ([]byte, error)
```

Comment:
This deserves a Changelog entry in my eyes.

Reply:
Indeed. Also need to use this soon, so added one as a new release for tomorrow.